### DISCUSSION TITLE
「エラーにより投稿できませんでした。何度も続く場合はサポートにお問い合わせください。」による投稿失敗

### DISCUSSION THREAD

**Discussion by k650250** [increments/qiita-discussions#885] [要望・アイデア・不具合]
記事を投稿しようとすると「エラーにより投稿できませんでした。何度も続く場合はサポートにお問い合わせください。」という赤文字のメッセージが表示され、投稿に失敗します。何度試しても同じ現象が続きます。下書きの保存は可能です。解決策をご教示願います。

投稿予定の記事は
https://qiita.com/drafts/12b5692f9e0ca96997fb
です。

**Comment by tanaka-asami415** [comment]
@k650250 
お問合せありがとうございます。
クローズになっているため既に解決済みかもしれませんが、もし現在もお困りのことがありましたら、一度 [限定共有投稿](https://help.qiita.com/ja/articles/qiita-private-article) を行っていただくか、画像等を添付いただけると幸いです🙇
